### PR TITLE
not sure why it'd be valuable to have this flexibility.

### PR DESCRIPTION
**Pull request by longrunningprocess**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/web-languageforge/911)

### CODE REVIEW

**Review by megahirt — CHANGES_REQUESTED** [review]
Looking great - I have one suggestion which I'll make in Github.

> Reviewed 2 of 2 files at r1.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-languageforge/911#-:-MY-I4mOC9-WUNwhbKq-:b3mej3g)* status: all files reviewed, 2 unresolved discussions (waiting on @longrunningprocess)

___
*[src/Site/Controller/Base.php, line 28 at r1](https://reviewable.io/reviews/sillsdev/web-languageforge/911#-MY-HYCc8tq8Wpkn-elt:-MY-HYCc8tq8Wpkn-elu:b-fva8dy) ([raw file](https://github.com/sillsdev/web-languageforge/blob/11ad60c3eb0dcc199dd737ae7c4e56b82f49b3f6/src/Site/Controller/Base.php#L28)):*
> ```PHP
>         $this->data['userName'] = '';
>         $this->data['version'] = VERSION;
>         $this->data['useMinifiedJs'] = false;
> ```

Unfortunately, this variable is still used in `container.html.twig` see line 25.  We could eventually get rid of this is we spent more time untangling our checked-in dependencies / those that are not bundled.  Ugh.

I will suggest a change in github, not reviewable.
___
*[src/Site/Controller/Base.php, line 127 at r1](https://reviewable.io/reviews/sillsdev/web-languageforge/911#-MY-Gz-sDvpOPSoqnO9A:-MY-Gz-t7SdurzM2i5qQ:brd89e4) ([raw file](https://github.com/sillsdev/web-languageforge/blob/6b5a12e0cd36148076784927d5aaf2a1eeffd820/src/Site/Controller/Base.php#L127)):*
> ```PHP
> 
>         $this->populateHeaderMenuViewdata();
>         $this->data['useCdn'] = USE_CDN;
> ```

We can do one step further - I searched through the code and cannot find a single usage of this variable.  Go ahead and delete the line.  Nice find!  An unused variable.

File: src/Site/Controller/Base.php [hunk 1]
```diff
@@ -124,7 +124,7 @@ protected function renderPage(Application $app, $viewName) {
         $this->data['themeColor'] = '#0a2440';
 
         $this->populateHeaderMenuViewdata();
-        $this->data['useCdn'] = USE_CDN;
+        $this->data['useCdn'] = false;
```
Comment (megahirt) [inline src/Site/Controller/Base.php:127]: Delete this line
```suggestion
```

File: src/Site/Controller/Base.php [hunk 2]
```diff
@@ -25,7 +25,7 @@ public function __construct() {
         $this->data['smallAvatarUrl'] = '';
         $this->data['userName'] = '';
         $this->data['version'] = VERSION;
-        $this->data['useMinifiedJs'] = USE_MINIFIED_JS;
+        $this->data['useMinifiedJs'] = false;
```
Comment (megahirt) [inline src/Site/Controller/Base.php:28]: Note you also need the `use Sil\PhpEnv\Env; // https://github.com/silinternational/php-env#class-env-summary-of-functions` at the top.  Github won't let me suggest that edit :(
I actually don't know if this is totally correct PHP syntax YMMV.
```suggestion
        $this->data['useMinifiedJs'] = (Env::requireEnv('ENVIRONMENT') == 'production');
```

**Review by megahirt — CHANGES_REQUESTED** [review]
suggestions based upon my reviewable.io review

**Review by longrunningprocess — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-languageforge/911)* status: 1 of 2 files reviewed, 4 unresolved discussions (waiting on @megahirt)

___
*[src/Site/Controller/Base.php, line 28 at r1](https://reviewable.io/reviews/sillsdev/web-languageforge/911#-MY-HYCc8tq8Wpkn-elt:-MY6SCVi8pJsptXYPEna:b-896fix) ([raw file](https://github.com/sillsdev/web-languageforge/blob/11ad60c3eb0dcc199dd737ae7c4e56b82f49b3f6/src/Site/Controller/Base.php#L28)):*
<details><summary><i>Previously, megahirt (Christopher Hirt) wrote…</i></summary><blockquote>

Unfortunately, this variable is still used in `container.html.twig` see line 25.  We could eventually get rid of this is we spent more time untangling our checked-in dependencies / those that are not bundled.  Ugh.

I will suggest a change in github, not reviewable.
</blockquote></details>

Done.
___
*[src/Site/Controller/Base.php, line 28 at r1](https://reviewable.io/reviews/sillsdev/web-languageforge/911#-MXhd6wUCJIe3D-OCAnx-r1-28:-MY6SDtH8oJXY2GVo54P:bb0wfrs) ([raw file](https://github.com/sillsdev/web-languageforge/blob/11ad60c3eb0dcc199dd737ae7c4e56b82f49b3f6/src/Site/Controller/Base.php#L28)):*
<details><summary><i>Previously, megahirt (Christopher Hirt) wrote…</i></summary><blockquote>

Note you also need the `use Sil\PhpEnv\Env; // https://github.com/silinternational/php-env#class-env-summary-of-functions` at the top.  Github won't let me suggest that edit :(
I actually don't know if this is totally correct PHP syntax YMMV.
```suggestion
        $this->data['useMinifiedJs'] = (Env::requireEnv('ENVIRONMENT') == 'production');
```
</blockquote></details>

lmk if you think looks ok, it's passing tests locally so I think that's good syntax.
___
*[src/Site/Controller/Base.php, line 127 at r1](https://reviewable.io/reviews/sillsdev/web-languageforge/911#-MY-Gz-sDvpOPSoqnO9A:-MY6SJZX4y06G75odLxJ:b-896fix) ([raw file](https://github.com/sillsdev/web-languageforge/blob/6b5a12e0cd36148076784927d5aaf2a1eeffd820/src/Site/Controller/Base.php#L127)):*
<details><summary><i>Previously, megahirt (Christopher Hirt) wrote…</i></summary><blockquote>

We can do one step further - I searched through the code and cannot find a single usage of this variable.  Go ahead and delete the line.  Nice find!  An unused variable.
</blockquote></details>

Done.
___
*[src/Site/Controller/Base.php, line 127 at r1](https://reviewable.io/reviews/sillsdev/web-languageforge/911#-MXhd6wUCJIe3D-OCAnx-r1-127:-MY6SKEG2l-IOC8L2xk0:b-896fix) ([raw file](https://github.com/sillsdev/web-languageforge/blob/11ad60c3eb0dcc199dd737ae7c4e56b82f49b3f6/src/Site/Controller/Base.php#L127)):*
<details><summary><i>Previously, megahirt (Christopher Hirt) wrote…</i></summary><blockquote>

Delete this line
```suggestion
```
</blockquote></details>

Done.

**Review by megahirt — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-languageforge/911#-:-MY9b9G62snfNTY4xF3q:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @longrunningprocess)

**Review by longrunningprocess — COMMENTED** [review]
> Reviewed 2 of 2 files at r3.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-languageforge/911)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @longrunningprocess)

**Pull request merged**